tests/suites/test_suite_ecp.function

```diff
@@ -1433,3 +1433,69 @@ exit:
     mbedtls_free(p);
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_TEST_HOOKS */
+void ecp_mod_mul_inv(char *input_A, int id, int ctype, int iret)
+{
+    int ret;
+    mbedtls_mpi_mod_modulus m;
+    mbedtls_mpi_mod_modulus_init(&m);
+    mbedtls_mpi_mod_residue A = { NULL, 0 };
+    mbedtls_mpi_mod_residue A_inverse = { NULL, 0 };
+    mbedtls_mpi_mod_residue rX = { NULL, 0 };
+    mbedtls_mpi_uint *rX_raw = NULL;
+    mbedtls_mpi_uint *p_inverse = NULL;
+    mbedtls_mpi_uint *p = NULL;
```

Comment:
Assuming the input is `void ecp_mod_mul_inv(char *input_A, int id, int ctype, int iret)`
This should be named as `mbedtls_mpi_uint *A = NULL;`

Generally when we are using an `mbedtls_mpi_uint *` pointer which is read through `mbedtls_test_read_mpi_core` the letter is kept.

    char *input_N ->  mbedtls_mpi_uint *N = NULL;

The inverse can be whatever the name of the pointer is plus  _inverse or _inv

Then the pointer for the raw data can be named as such

`mbedtls_mpi_uint *p_inverse = NULL;` --> `mbedtls_mpi_uint *rA_inv_raw= NULL;`

Reply:
Just renamed the data, and for this variable, not very the smae, ` mbedtls_mpi_uint *p_inverse = NULL;` --> `mbedtls_mpi_uint *A_inv= NULL;`